MVP/defaults-build.yaml

```diff
@@ -0,0 +1,74 @@
+- defaults:
+    name: defaults-build
+    description: |
+        Managed by Jenkins Job Builder. Do not edit via web.
+    node: 'jslave-platform-rhel7' #[MVP] please replace this value with jenkins slave name/label of your own.
+    #concurrent: true
+    parameters:
+        - string:
+            name: id
+        - string:
+            name: task_id
+        - string:
+            name: CI_TYPE
+        - string:
+            name: CI_MESSAGE
+        - string:
+            name: tag 
+        - string:
+            name: name
+        - string:
+            name: version
+        - string:
+            name: release
```

Comment:
This is defined by the message in the trigger - you used CI_TYPE = 'brew-tag' there, which is indeed only related to tagging task in Brew. This task is automatically done as a subtask of every non-scratch build (the finished build is placed into the *destination tag* for a given target). This trigger also imply the following:
1) The trigger will not fire after a scratch build is finished
2) The trigger will fire even after someone manually tags an old build, totally unrealted to any build task.

IMO, our default trigger should be on any build task (both official and scratch ones).